### PR TITLE
Minimal updates for LQOI 0.6.0

### PR DESCRIPTION
**Pull request by ianfiske**
Addresses #49 .  I made the minimal changes necessary to work with LQOI 0.6.0 and have the tests pass.

### CODE REVIEW

File: src/MOIWrapper.jl [hunk 1]
```diff
@@ -82,6 +82,19 @@ function replace_inf(x::Real)
...
 
+
+function LQOI.get_objectivesense(model::Optimizer)
+    s = get_obj_sense(model.inner)
+    if s == 1.0
+        return MOI.MIN_SENSE
+    elseif s == -1.0
+        return MOI.MAX_SENSE
+    else
+        error("Invalid objective sense: $(sense)")
```
Comment (odow) [inline src/MOIWrapper.jl]: `$(s)` not `$(sense)`.

Comment (ianfiske) [reply src/MOIWrapper.jl]: good catch -- fixed.

File: src/MOIWrapper.jl [hunk 2]
```diff
@@ -82,6 +82,19 @@ function replace_inf(x::Real)
...
 
+
+function LQOI.get_objectivesense(model::Optimizer)
+    s = get_obj_sense(model.inner)
+    if s == 1.0
+        return MOI.MIN_SENSE
+    elseif s == -1.0
+        return MOI.MAX_SENSE
+    else
+        error("Invalid objective sense: $(s)")
```
Comment (odow) [inline src/MOIWrapper.jl]: Is it possible for `get_obj_sense` to return something else? Can we add a test?

Comment (blegat) [reply src/MOIWrapper.jl]: Couldn't it be `MOI.FEASIBILITY_SENSE` ?

Comment (mlubin) [reply src/MOIWrapper.jl]: According to https://projects.coin-or.org/Clp/browser/trunk/Clp/src/Clp_C_Interface.h#L419, it looks like a return value of 0.0 maps to `MOI.FEASIBILITY_SENSE` (ignore the objective).

Comment (ianfiske) [reply src/MOIWrapper.jl]: Thank you for catching that and bonus for pointing me to the location in the Clp source!  Just pushed a fix.

@odow I briefly looked add the idea of adding a test, but seems that the tests are owned by MOI -- would that be the place to add such a test?

Comment (odow) [reply src/MOIWrapper.jl]: Just add a new test to the bottom of https://github.com/JuliaOpt/Clp.jl/blob/master/test/MOIWrapper.jl.
Create a model, set the objective sense to -3, and then query `LQOI.get_objective_sense`. Alternatively, just change the `elseif` prior to the `FEASIBILITY_SENSE` into an `else` and remove the error.

Comment (ianfiske) [reply src/MOIWrapper.jl]: Thank @odow -- I just pushed a change with your alternative suggestion.

File: src/MOIWrapper.jl [hunk 3]
```diff
@@ -82,6 +82,19 @@ function replace_inf(x::Real)
...
 
+
+function LQOI.get_objectivesense(model::Optimizer)
+    s = get_obj_sense(model.inner)
+    if s == 1.0
+        return MOI.MIN_SENSE
+    elseif s == -1.0
+        return MOI.MAX_SENSE
+    else
+        return MOI.FEASIBILITY_SENSE
```
Comment (blegat) [inline src/MOIWrapper.jl]: I would add `@assert iszero(s)` before the return just to be sure

Comment (ianfiske) [reply src/MOIWrapper.jl]: Thanks @blegat -- just added that.

**Comment by mlubin** [comment]
Thanks! I'll tag right now.

**Pull request merged**